File: add/codesnippet/VisualBasic/31379e48-984f-4b70-8613-_1.vb
    Private Shared Sub ToFromXmlDemo()
        Dim memberAccessPerm As New ReflectionPermission(ReflectionPermissionFlag.MemberAccess)
        Dim restrictedMemberAccessPerm As New ReflectionPermission(PermissionState.None)
        restrictedMemberAccessPerm.FromXml(memberAccessPerm.ToXml())
        Console.WriteLine("Result of ToFromXml = " + restrictedMemberAccessPerm.ToString())

    End Sub 'ToFromXmlDemo
End Class 'ReflectionPermissionDemo

File: add/codesnippet/CSharp/31379e48-984f-4b70-8613-_1.cs
    private static void ToFromXmlDemo()
    {
        ReflectionPermission memberAccessPerm = new ReflectionPermission(ReflectionPermissionFlag.MemberAccess);
        ReflectionPermission restrictedMemberAccessPerm = new ReflectionPermission(PermissionState.None);
        restrictedMemberAccessPerm.FromXml(memberAccessPerm.ToXml());
        Console.WriteLine("Result of ToFromXml = " +
            restrictedMemberAccessPerm.ToString());
    }